SwaprHQ/swapr-dapp
# [SWA-128][FEAT] - Update social links, links to Range Liquidity and new Popup

## Fixes: [SWA-128](https://linear.app/swaprhq/issue/SWA-128/update-social-links-and-showcase-new-amm-concentrated-liq)

# Description

* Update Discords links
* Remove $SWPR ClaimModal component
* Add new Range Liquidity header link
* Add new Range Liquidity popup component

# Visual evidence
![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/9e2bcb2e-f732-4a77-8659-ddad45856555)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) Connect your wallet
5) The new popup and header item should be visible, and they should redirect the user to `https://swapr.liquidity.eth.limo/`, all Discord links should send a correct invitation to Swapr Discord and the Claim modal to claim $SWPR should be unavailable
